docs.md

```diff
@@ -1112,19 +1065,19 @@ function EmailInput ({
 
 ## LabeledField
 
-A fieldset wrapper for redux-form controlled inputs. This wrapper adds a label component (defaults to [InputLabel][71])
-above the wrapped component and an error component below (defaults to [InputError][68]). Additionally, it adds the class `"error"`
-to the fieldset if the input is touched and invalid.
+A container wrapper for redux-form controlled inputs. This wrapper adds a label component (defaults to [InputLabel][68])
+above the wrapped component and an error component below (defaults to [InputError][65]). Additionally, it adds the class `"error"`
+to the container if the input is touched and invalid.
 
 In order to populate the `InputLabel` and `InputError` correctly, you should pass all the props of the corresponding input
 to this component. To prevent label-specific props from being passed to the input itself,
-use the [omitLabelProps][91] helper.
+use the [omitLabelProps][88] helper.
 
 ### Parameters
 
-*   `hideErrorLabel` **[Boolean][144]?** A boolean determining whether to hide the error label on input error (optional, default `false`)
-*   `labelComponent` **[Function][143]** A custom label component for the input (optional, default `InputLabel`)
-*   `errorComponent` **[Function][143]** A custom error component for the input (optional, default `InputError`)
+*   `hideErrorLabel` **[Boolean][141]?** A boolean determining whether to hide the error label on input error (optional, default `false`)
+*   `labelComponent` **[Function][140]** A custom label component for the input (optional, default `InputLabel`)
+*   `errorComponent` **[Function][140]** A custom error component for the input (optional, default `InputError`)
```

Comment:
Hmm, I would expect `as` to show up here. Maybe it's not showing up because JSDoc doesn't respect "Element Type" as a type? I don't think we can assume the naming convention in the prop types library maps 1:1 to JSDoc